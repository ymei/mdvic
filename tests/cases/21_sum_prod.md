Inline sums: $\sum_{i=1}^{n} i$ and $\prod_{j=1}^{3} j$.

Display:
$$\sum_{k=0}^{m} \frac{k}{k+1}$$

