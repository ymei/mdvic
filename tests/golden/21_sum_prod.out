Inline sums: ∑i₌₁ⁿ i and ∏j₌₁³ j.

Display: 
m
∑
k=0
k
---
k+1
